LibreNMS/Util/ModuleTestHelper.php

```diff
@@ -487,7 +487,7 @@ private function cleanSnmprecData(&$data)
         foreach ($private_oid as $oid) {
             if (isset($data[$oid])) {
                 $parts = explode('|', $data[$oid], 3);
-                $parts[2] = '<private>';
+                $parts[2] = $parts[1] === '4' ? '<private>' : '3C707269766174653E';
```

Comment:
I would change this to `$parts[1] == '4x' ? '3C707269766174653E' : '<private>'` but I'm not sure it makes a difference.